hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/storage/HoodieParquetWriter.java

```diff
@@ -40,7 +39,7 @@
  * HoodieParquetWriter extends the ParquetWriter to help limit the size of underlying file. Provides a way to check if
  * the current file can take more records with the <code>canWrite()</code>
  */
-public class HoodieParquetWriter<T extends HoodieRecordPayload, R extends IndexedRecord>
+public class HoodieParquetWriter<R extends IndexedRecord>
```

Comment:
Why do we need to change this class?

Reply:
> Why do we need to change this class?

The Generic "T" is useless in this class, and it causes some generic problems in the abstraction, so I removed it.